app/Services/Company/Adminland/Employee/ImportEmployeesFromTemporaryTable.php

```diff
@@ -71,14 +87,29 @@ private function import(): void
                         'last_name' => $report->employee_last_name,
                         'permission_level' => 300,
                         'send_invitation' => false,
-                    ])->onQueue('low');
```

Comment:
Why do you remove this?

We NEED to continue indicate different queues. Once we will have the flows feature, we will have so much queues all the time, and we need to indicate which service has priority over other services.

Reply:
Indeed, that's exactly what I would like: using the 'low' queue for not important jobs, and the 'default' queue for the rest, like this action.
Actually right now *ALL* existing jobs are using the 'low' queue, which is useless ...

Reply:
@asbiin yeah but do we have a way to control the queue names with Fortrabbit and set priority?

Reply:
@asbiin also, yeah, all existing jobs are using the `low` queue, and this is because there are currently no job (except this one) that should be `high`. But we will need to have high priority queue.

Reply:
The default queue is named "default", which is fine by me.
And yes we can specify the queue to run, RTFM: https://laravel.com/docs/8.x/queues#specifying-the-connection-queue